### DISCUSSION TITLE
How to customize minio UI

### DISCUSSION THREAD

**Discussion by pedrrcorr** [minio/minio#20695] [Q&A]
Hello everyone, 

This is the actually minio consule ui,

![miniouichanges](https://github.com/user-attachments/assets/7f471852-df31-4878-b8ee-75307c9a93b9)

I need to change the object metadata viewport, to highlight and zoom the metadata viewport, but keeping the majority of the current user interface (UI).

I don't need the code to do it. I am inquiring whether it is feasible and if it is recommended and scalable.

Thank you everyone for helping me.

**Comment by marktheunissen** [comment]
Yes it's possible, because the console is open source: https://github.com/minio/console you can modify and maintain your patches.

If the patch is good and useful for all users and customers, make a pull request and we might merge it to the main product.

Another option is some kind of browser plugin, or JavaScript snippet that can simply run client side and do the small modifications you need.
